lib/engine/game/g_1849/game.rb

```diff
@@ -532,13 +573,25 @@ def check_distance(route, _visits)
         end
 
         def check_other(route)
+          check_track_type(route)
           return unless (route.stops.map(&:hex).map(&:id) & PORT_HEXES).any?
 
           raise GameError, 'Route must include two non-port stops.' unless route.stops.size > 2
         end
 
+        def check_track_type(route)
```

Comment:
Let's come up with a more descriptive method name. Something like `check_e_train_track`.

Reply:
Changed to check_e_train_track_type. A little wordy, but clearer.